pkg/media/audio_smear.go

```diff
@@ -265,13 +260,11 @@ func JoinAudioVideo(ctx context.Context, seg *SegmentData, output io.Writer) err
 	errCh := make(chan error)
 	go func() {
 		err := HandleBusMessages(ctx, pipeline)
-		cancel()
```

Comment:
Most of these we do need - the bus handler setting down is actually the primary sign from the gstreamer pipeline that we're done and should start shutting down